packages/expression-parser/src/expression-parser/ExpressionParser.js

```diff
@@ -68,6 +60,20 @@ export class ExpressionParser {
     this.validExpressionCache = new Set();
   }
 
+  /**
+   * @param {string} name
+   * @return bool
+   */
+  isBuiltInFunction(name) {
+    try {
+      // see https://mathjs.org/docs/reference/functions/help.html
+      this.math.help(name);
```

Comment:
Wow, I hadn't realised mathjs has `help` built into it! I wonder if you can extend the default functions with your own ones, could neat to provide in-app help in viz builder or something...